src/components/signatures/signTypedDataV3-sign.js

```diff
@@ -0,0 +1,176 @@
+import { recoverTypedSignature } from '@metamask/eth-sig-util';
+import { toChecksumAddress } from 'ethereumjs-util';
+import globalContext from '../..';
+import { EIP712Domain } from '../../signatures/utils';
+
+export function signTypedDataV3Component(parentContainer) {
+  parentContainer.insertAdjacentHTML(
+    'beforeend',
+    `<div class="col-xl-4 col-lg-6 col-md-12 col-sm-12 col-12 d-flex align-items-stretch">
```

Comment:
This is great! Would it make sense to keep a separate HTML file for each component so that it has syntax highlighting, and import the markup here?

Reply:
Yes. There are other ways to do it too, like importing a JSX parser so you can use JSX fragments instead of strings.

Reply:
(There's also [tricks to making this work](https://dev.to/boyum/using-tagged-template-literals-to-syntax-highlight-code-in-strings-4h1i), or just bit the bullet and port to React :) )